pom.xml

```diff
@@ -40,63 +40,44 @@
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
 
-        <spring.security.version>5.3.5.RELEASE</spring.security.version>
-        <spring.cloud.openfeign.version>2.2.5.RELEASE</spring.cloud.openfeign.version>
-        <openfeign.version>10.10.1</openfeign.version>
+        <spring-boot-dependencies.version>2.6.2</spring-boot-dependencies.version>
+        <spring-cloud-dependencies.version>2021.0.0</spring-cloud-dependencies.version>
     </properties>
 
     <dependencies>
-        <dependency>
-            <groupId>org.springframework.security</groupId>
-            <artifactId>spring-security-config</artifactId>
-            <version>${spring.security.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.springframework.security</groupId>
-            <artifactId>spring-security-core</artifactId>
-            <version>${spring.security.version}</version>
-        </dependency>
         <dependency>
             <groupId>org.springframework.security</groupId>
             <artifactId>spring-security-oauth2-client</artifactId>
-            <version>${spring.security.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.security</groupId>
-            <artifactId>spring-security-oauth2-jose</artifactId>
-            <version>${spring.security.version}</version>
-        </dependency>
-
         <dependency>
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-openfeign-core</artifactId>
-            <version>${spring.cloud.openfeign.version}</version>
         </dependency>
-
         <dependency>
             <groupId>io.github.openfeign</groupId>
             <artifactId>feign-core</artifactId>
-            <version>${openfeign.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>io.github.openfeign</groupId>
-            <artifactId>feign-slf4j</artifactId>
-            <version>${openfeign.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>io.github.openfeign</groupId>
-            <artifactId>feign-hystrix</artifactId>
-            <version>${openfeign.version}</version>
-        </dependency>
-
-        <dependency>
-            <groupId>org.projectlombok</groupId>
-            <artifactId>lombok</artifactId>
-            <version>1.18.16</version>
-            <scope>provided</scope>
         </dependency>
     </dependencies>
 
+    <dependencyManagement>
```

Comment:
I'd like to not have this project, either in actuality or impliciation, related to spring boot directly. it can be used without it. would you consider using the BOM's for spring security and spring cloud openfeign as alternatives?

```
        <spring-security.version>5.6.1</spring-security.version>
        <spring-cloud-openfeign.version>3.1.0</spring-cloud-openfeign.version>
```
...
```
    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>org.springframework.security</groupId>
                <artifactId>spring-security-bom</artifactId>
                <version>${spring-security.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
            <dependency>
                <groupId>org.springframework.cloud</groupId>
                <artifactId>spring-cloud-openfeign-dependencies</artifactId>
                <version>${spring-cloud-openfeign.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>
```

Reply:
For the case where this library is used **without** spring-boot, the `spring-cloud-openfeign-core` dependency has a transient dependency on spring-boot and spring-boot-starter so this project depends on a version of spring-boot anyway (run the maven goal dependency:tree to see the details). 
For the case where this library is used with spring-boot, specifically a different version to the one in this library, one would have to configure the dependency on this library to exclude the three dependencies defined this this pom file. Not complicated, so maybe this is the route to take.
My most compelling reason for using the spring-boot-dependencies pom and the cloud one is that spring-boot releases a new version every 3 to 4 weeks (spring-cloud every 2 or 3 months) and it's very easy to release "compatible" updates to this library by simply changing the spring-boot-dependencies version once every 3 to 4 weeks, and the cloud version 3 or 4 times a year - it's much easier than digging through the dependency tree the way I am right now :-).

Reply:
Interesting. okay. given that i think its fine to use the boot and cloud dependencies.